pkg/controller/nodes/subworkflow/launchplan.go

```diff
@@ -18,15 +20,36 @@ type launchPlanHandler struct {
 	launchPlan launchplan.Executor
 }
 
+func fetchParentNodeExecutionID(nCtx handler.NodeExecutionContext) (*core.NodeExecutionIdentifier, error) {
```

Comment:
can you rename? fetch kind of implies some network call...

Reply:
get?